modules/tinymce/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Improved
+- Added background and additional spacing for the text labeled buttons in the toolbar to impove visual clarity #TINY-8617
```

Comment:
As this is slated for 6.0.2, this must be a bug fix and can't be an improvement. So we need to move this and reword it.